src/pom.xml

```diff
@@ -10,16 +7,27 @@
     <version>0.0.1-SNAPSHOT</version>
     <packaging>pom</packaging>
     <name>SHOGun2</name>
+    <description>SHOGun2 is a Java based WebGIS framework</description>
+    <url>https://github.com/terrestris/shogun2</url>
 
     <organization>
-        <name>terrestris GmbH &#038; Co. KG</name>
+        <name>terrestris GmbH &amp; Co. KG</name>
         <url>http://www.terrestris.de/</url>
     </organization>
 
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
     <scm>
-        <connection>scm:git:git://github.com/terrestris/shogun2.git</connection>
-        <developerConnection>scm:git:[fetch=]git://github.com/terrestris/shogun2.git[push=]git@github.com:terrestris/shogun2.git</developerConnection>
-        <url>https://github.com/terrestris/shogun2.git</url>
+        <url>https://github.com/dnlkoch/shogun2</url>
+        <connection>scm:git:git://github.com/dnlkoch/shogun2.git</connection>
+        <developerConnection>scm:git:git@github.com:dnlkoch/shogun2.git</developerConnection>
```

Comment:
Only needed for the moment, otherwise the terrestris repository would be manipulated. I would like to prevent this while testing.